tests/mentors/test_get_mentors.py

```diff
@@ -0,0 +1,92 @@
+from rest_framework.test import APITestCase, APIClient
+from rest_framework import status
+from mentors.models import Mentor
+
+
+class MentorsListTestCase(APITestCase):
+    def setUp(self):
+        """
+        Create different mentors with different years and flag input in test database
+        """
+        self.mentor1 = Mentor.objects.create(
+            id=1,
+            name="test1",
+            email="test@tmail.com",
+            contact="0000000000",
+            detail="test_detail",
+            description="test_desc",
+            year=2019,
+            flag=True,
+        )
+        self.mentor2 = Mentor.objects.create(
+            id=2,
+            name='test2',
+            email="test2@tmail.com",
+            contact="1111111111",
+            detail="test_detail",
+            description="test_desc",
+            year=2019,
+            flag=True,
+        )
+        self.mentor3 = Mentor.objects.create(
+            id=3,
+            name='test3',
+            email="test3@tmail.com",
+            contact="2222222222",
+            detail="test_detail",
+            description="test_desc",
+            year=2019,
+            flag=False,
+        )
+        self.mentor4 = Mentor.objects.create(
+            id=4,
+            name='test4',
+            email="test4@tmail.com",
+            contact="3333333333",
+            detail="test_detail",
+            description="test_desc",
+            year=2020,
+            flag=True,
+        )
+
+    def test_pass_get_mentors_many(self):
+        """
+        Test for mulitple mentors
+        """
+        get_group_api = "/mentors/list/2019/"
+        get_client = APIClient()
+        response = get_client.get(get_group_api)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_pass_get_mentors_single(self):
+        """
+        Test for single mentor
+        """
+        get_group_api = "/mentors/list/2020/"
+        get_client = APIClient()
+        response = get_client.get(get_group_api)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
What's the point of testing single mentor list if we are already testing multiple mentor test? @akhil-maker

Reply:
Actually I thought if I am adding test for multiple and empty, then why not for single one of a different year. So, added.

Reply:
Um.. I think it's pretty redundant because you are testing for multiple users in the year 2019 and no users for 2021, so I don't think this is needed here. Any suggestions @anshsrtv sir?

Reply:
Okay! Let's see the suggestions from sir.

Reply:
@sahil9001 With the recent commits, it does make sense now, and test cases can be redundant, keeping in mind, the objectives of testing. ✌️ 

@akhil-maker please check the status codes, as well, in all the test cases.🙌

Reply:
Okay then !

Reply:
@anshsrtv I will add the code to check status codes soon.